files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md

```diff
@@ -308,10 +305,10 @@ curl https://developer.mozilla.org/docs/Web/API/fetch -L -I | grep location | aw
 最终的输出应该是这样的
 
 ```bash
-https://developer.mozilla.org/en-US/docs/Web/API/fetch
-https://developer.mozilla.org/en-US/docs/Web/API/GlobalFetch/GlobalFetch.fetch()
-https://developer.mozilla.org/en-US/docs/Web/API/GlobalFetch/fetch
-https://developer.mozilla.org/en-US/docs/Web/API/fetch
+https://developer.mozilla.org/zh-CN/docs/Web/API/fetch
+https://developer.mozilla.org/zh-CN/docs/Web/API/GlobalFetch/GlobalFetch.fetch()
+https://developer.mozilla.org/zh-CN/docs/Web/API/GlobalFetch/fetch
+https://developer.mozilla.org/zh-CN/docs/Web/API/fetch
```

Comment:
Same as the previous one :)

```suggestion
https://developer.mozilla.org/en-US/docs/Web/API/fetch
```

![image](https://github.com/mdn/translated-content/assets/15844309/18345be7-45a6-4a91-b6dd-308f048b3e68)